src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -21,43 +21,43 @@
 
 public class ModelTest extends SimulationTest {
 
-    int population;
-    int nInfections;
-    int nIter;
-    int nDays;
-    int RNGSeed;
-
-    @Before
-    public void setupParams() {
-        population = 10000;
-        nInfections = 10;
-        nIter = 1;
-        nDays = 60;
-        RNGSeed = 402;
-    }
+    private static int population = 10000;
+    private static int nInfections = 150;
+    private static int nIter = 1;
+    private static int nDays = 60;
+    private static int RNG = 402;

```

Comment:
I really wanted to remove it fully (since I thought it would just being the seed that was being used in the general RNG), but this doesn't work. When I added it back in I just want with RNG for no particular reason other than forgetting the old name (and I'm also not set on it and happy to just call it "seed" or something!)